src/main/resources/application.properties

```diff
@@ -317,6 +317,11 @@ alpine.oidc.team.synchronization=false
 # will most likely need to be configured.
 alpine.oidc.teams.claim=groups
 
+# Optional
+# Define whether system requirement check is disable.
+# The default value is false.
+disable.system.requirement.check=false
```

Comment:
don't want to nitpick, but I suggest {{system.requirement.check.enabled=true}}, similar to the CORS setting etc.